OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -211,7 +211,10 @@ public interface INotifyInfiltration
 		void Infiltrating(Actor self);
 	}
 
-	public interface ITechTreePrerequisiteInfo : ITraitInfo { }
+	public interface ITechTreePrerequisiteInfo : ITraitInfo
+	{
+		string[] Prerequisites(ActorInfo info);
```

Comment:
Change this to `IEnumerable<string>` to allow both `string[]` and `HashSet<string>` to be returned.   Also needs a newline after the closing `}`.